specification/netapp/resource-manager/Microsoft.NetApp/stable/2021-08-01/netapp.json

```diff
@@ -5004,6 +5012,11 @@
           "description": "Snapshot policy Properties",
           "$ref": "#/definitions/snapshotPolicyProperties",
           "x-ms-client-flatten": true
+        },
+        "systemData": {
+          "readOnly": true,
+          "description": "The system meta data relating to this resource.",
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
         }
```

Comment:
consider using the commontypes [https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/types.json](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/types.json)

Reply:
sorry I don't get your point, `systemData` is being referenced from common types.

Reply:
the common type Resource has common fields such as id, name, type and systemData

Reply:
Are you suggesting we change all our tracked resources to reference from common types? 
That does pose a bit of "problem" in validation in the pipeline as `Location` will be flagged for `PATCH` operations in those cases it shares the same type. 
Should we then have a another type for patch operations or is there some other best practice for that scenario?

Reply:
@zizw123 @raosuhas  updated PR to reference common types